cli/template/base/next.config.js

```diff
@@ -17,6 +17,7 @@ const config = {
     locales: ["en"],
     defaultLocale: "en",
   },
+  transpilePackages: ["geist"],
```

Comment:
?

Reply:
I think there was an issue with Geist, but without this line there would be an issue when using the pages router. 
Check this issue: https://github.com/vercel/geist-font/issues/13